[2021-11-15 11:09:38] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\wamp64\www\prestashop\var\cache\prod\appDBF1.tmp" to "C:\wamp64\www\prestashop\var\cache\prod/appProdProjectContainerUrlMatcher.php"." at C:\wamp64\www\prestashop\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\wamp64\\www\\prestashop\\var\\cache\\prod\\appDBF1.tmp\" to \"C:\\wamp64\\www\\prestashop\\var\\cache\\prod/appProdProjectContainerUrlMatcher.php\". at C:\\wamp64\\www\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2021-11-15 11:09:38] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\wamp64\www\prestashop\var\cache\prod\appDCE0.tmp" to "C:\wamp64\www\prestashop\var\cache\prod/appProdProjectContainerUrlMatcher.php"." at C:\wamp64\www\prestashop\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\wamp64\\www\\prestashop\\var\\cache\\prod\\appDCE0.tmp\" to \"C:\\wamp64\\www\\prestashop\\var\\cache\\prod/appProdProjectContainerUrlMatcher.php\". at C:\\wamp64\\www\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2021-11-15 11:09:38] request.CRITICAL: Exception thrown when handling an exception (RuntimeException: Unable to create the cache directory (C:\wamp64\www\prestashop\var\cache\prod/twig/50). at C:\wamp64\www\prestashop\vendor\twig\twig\src\Cache\FilesystemCache.php line 57) {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (C:\\wamp64\\www\\prestashop\\var\\cache\\prod/twig/50). at C:\\wamp64\\www\\prestashop\\vendor\\twig\\twig\\src\\Cache\\FilesystemCache.php:57)"} []
[2021-11-15 11:39:37] request.INFO: Matched route "admin_emails_send_test". {"route":"admin_emails_send_test","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\EmailController::sendTestAction","_legacy_controller":"AdminEmails","_legacy_link":"AdminEmails:testEmail","_route":"admin_emails_send_test"},"request_uri":"http://localhost/prestashop/admin3502epnae/index.php/configure/advanced/emails/send-testing-email?_token=EQjR_evIS129KgcJuwXG-LQ9jYYB0mb4TeweLpLdQRA","method":"POST"} []
[2021-11-15 11:39:37] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-11-15 11:39:37] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"pierre.hardy1999@gmail.com"} []
[2021-11-15 11:39:38] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Serializer\Exception\NotEncodableValueException: "Malformed UTF-8 characters, possibly incorrectly encoded" at C:\wamp64\www\prestashop\vendor\symfony\symfony\src\Symfony\Component\Serializer\Encoder\JsonEncode.php line 51 {"exception":"[object] (Symfony\\Component\\Serializer\\Exception\\NotEncodableValueException(code: 0): Malformed UTF-8 characters, possibly incorrectly encoded at C:\\wamp64\\www\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Serializer\\Encoder\\JsonEncode.php:51)"} []
[2021-11-15 11:39:38] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-11-15 11:40:17] request.INFO: Matched route "admin_system_information_check_files". {"route":"admin_system_information_check_files","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\SystemInformationController::displayCheckFilesAction","_legacy_controller":"AdminInformation","_legacy_link":"AdminInformation:checkFiles","_route":"admin_system_information_check_files"},"request_uri":"http://localhost/prestashop/admin3502epnae/index.php/configure/advanced/system-information/files?_token=EQjR_evIS129KgcJuwXG-LQ9jYYB0mb4TeweLpLdQRA","method":"POST"} []
[2021-11-15 11:40:17] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2021-11-15 11:40:17] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"pierre.hardy1999@gmail.com"} []
[2021-11-15 11:42:17] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 120 seconds exceeded" at C:\wamp64\www\prestashop\src\Adapter\Requirement\CheckMissingOrUpdatedFiles.php line 70 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 120 seconds exceeded at C:\\wamp64\\www\\prestashop\\src\\Adapter\\Requirement\\CheckMissingOrUpdatedFiles.php:70)"} []
[2021-11-15 11:42:17] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-11-15 11:40:17] request.INFO: Matched route "admin_common_notifications". {"route":"admin_common_notifications","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\CommonController::notificationsAction","_legacy_controller":"AdminCommon","_legacy_link":"AdminCommon","_route":"admin_common_notifications"},"request_uri":"http://localhost/prestashop/admin3502epnae/index.php/common/notifications?_token=EQjR_evIS129KgcJuwXG-LQ9jYYB0mb4TeweLpLdQRA&rand=1636972817066","method":"POST"} []
[2021-11-15 11:42:17] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 120 seconds exceeded" at C:\wamp64\www\prestashop\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\Handler\StrictSessionHandler.php line 48 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 120 seconds exceeded at C:\\wamp64\\www\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\StrictSessionHandler.php:48)"} []
[2021-11-15 11:42:17] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("Failed to start the session: already started by PHP."). at C:\wamp64\www\prestashop\src\PrestaShopBundle\Resources\views\Admin\Exception\error.html.twig line 60) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Failed to start the session: already started by PHP.\"). at C:\\wamp64\\www\\prestashop\\src\\PrestaShopBundle\\Resources\\views\\Admin\\Exception\\error.html.twig:60, RuntimeException(code: 0): Failed to start the session: already started by PHP. at C:\\wamp64\\www\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:146)"} []
[2021-11-15 11:42:17] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 120 seconds exceeded" at C:\wamp64\www\prestashop\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\Handler\StrictSessionHandler.php line 48 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 120 seconds exceeded at C:\\wamp64\\www\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\StrictSessionHandler.php:48)"} []
[2021-11-15 11:42:17] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("Failed to start the session: already started by PHP."). at C:\wamp64\www\prestashop\src\PrestaShopBundle\Resources\views\Admin\Exception\error.html.twig line 60) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Failed to start the session: already started by PHP.\"). at C:\\wamp64\\www\\prestashop\\src\\PrestaShopBundle\\Resources\\views\\Admin\\Exception\\error.html.twig:60, RuntimeException(code: 0): Failed to start the session: already started by PHP. at C:\\wamp64\\www\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:146)"} []
